pkg/cli/debug/debug.go

```diff
@@ -62,6 +62,9 @@ const (
 	containerResourcesAnnotationPrefix = "resources.workload.openshift.io/"
 	// podWorkloadTargetAnnotationPrefix contains the prefix for the pod workload target annotation
 	podWorkloadTargetAnnotationPrefix = "target.workload.openshift.io/"
+	kubeOSNodeSelector                = "kubernetes.io/os"
+	commandLinuxShell                 = "/bin/sh"
+	commandWindowsShell               = "powershell"
```

Comment:
It is risky to use `powershell.exe`. In certain Windows container images, the binary is `pwsh.exe`. I suggest using [cmd.exe](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd) (the older command interpreter) which is guaranteed to exist in all Windows container images.